src/models/movie.js

```diff
@@ -0,0 +1,68 @@
+import Backbone from 'backbone';
+
+const Movie = Backbone.Model.extend({
+  urlRoot: 'http://localhost:3000/movies',
+  defaults: {
+
+  },
+  initialize(attributes) {
+  },
+  validate(attributes) {
+
+    const errors = {};
+    const title = this.get('title');
+    const releaseDate = this.get('release_date');
+
+    console.log(title);
+    console.log(`release date is ${releaseDate}`);
+
+    if (!title) {
+      console.log("Cannot add a movie without a name");
+      errors['title'] = ["Cannot add a movie without a name"];
+    }
+
+    if (!releaseDate) {
+      errors['releaseDate'] = ["Cannot add a move that does not have a release date"];
+    }
+
+    if ( Object.keys(errors).length > 0 ) {
+      return errors;
+    } else {
+      return false;
+    }
+  },
+
+  add(newMovie) {
+    if(!newMovie.isValid()){
+      $('.display-status').html('')
```

Comment:
This function seems a little strange to me. It's defined on Movie, which implies you're currently inside a movie object, but it requires you to pass in a separate movie object and saves that to the database. Instead, it would probably be cleaner to have this function save this movie to the API, and call it on the new movie instead of the old.

Alternatively, you could say it's called on the old movie, but internally does the work of creating a new movie and saving that (work that is currently done in `MovieView.add()`)